openpdf/src/main/java/com/lowagie/text/pdf/PdfContentByte.java

```diff
@@ -1435,7 +1437,15 @@ public void showText(String text) {
         showText2(text);
         content.append("Tj").append_i(separator);
     }
-
+    
+	// TIBCO Software #4 : Part 1 - START
```

Comment:
Is there any point in keeping comments like these when we are adopting their changes into the OpenPDF codebase?🤔